auth4genai/snippets/get-started/langchain-next-js/call-others-api.mdx

```diff
@@ -111,7 +111,7 @@ AUTH0_CLIENT_ID='<your-auth0-application-client-id>'
 AUTH0_CLIENT_SECRET='<your-auth0-application-client-secret>'
 
 AUTH0_AUDIENCE="https://your.domain.us.langgraph.app"
-AUTH0_SCOPE="openid profile email"
+AUTH0_SCOPE="openid profile email offline_access"
```

Comment:
Not needed for nextjs-auth0, because the default requested scopes include `offline_access`.
```suggestion
```